docs/methods/search_parameter_optimization.md

```diff
@@ -1,12 +1,36 @@
-# Search Parameter Optimization
+# Optimization and Calibration
+In peptide centric DIA search, calibration of the library and optimization of search parameters is required to maximize the number of confident identifications. AlphaDIA performs both calibration and optimization iteratively. Calibration removes the systematic deviation of observed and library values to account for technical variation from the LC or MS instrument. Optimization reduces the search space to improve the confidence in identifications and to accelerate search.
+:::{note}
+Calibration and optimization are different but connected to transfer learning. In [transfer learning](./transfer-learning.md) the residual (non-systematic) variation is learned and thereby reduced. This usually leads to better performance if used with optimization and calibration.
+:::
 
-## Calibration and optimization
+## Overview of optimization
+AlphaDIA can perform optimization for the following parameters, `target_ms1_tolerance`, `target_ms2_tolerance`, `target_mobility_tolerance` and `target_rt_tolerance`. There are two optimization strategies, targeted and automatic.
+
+### Targeted optimization
+The search space is progressively narrowed until a target tolerance is reached for a given parameter.
+
+To activate targeted optimization for example for fragment m/z tolerance, set `target_ms2_tolerance` to `10` for using a target tolerance of 10 ppm.
+For retention time, the target value can be either set as an absolute value in seconds or as a fraction of the total retention time range.
+
+For example, setting `target_rt_tolerance` to `300` will result in a target tolerance of 300 seconds, while setting it to `0.3` will use 30% of the gradient length as the target tolerance.
+
+### Automatic optimization
+In automatic optimization the search space is reduced until an optimal value is detected. This optimization is curently performed for every raw file individually. The results of the optimization can be found in the [stats.tsv](<project:../methods/output-format.md>) file in the output directory.
+
+To activate automatic optimization for example for retention time, set `target_rt_tolerance` to `0.0`.
```

Comment:
(nit) To activate automatic optimization for a certain quantity, set the respective target to `0.0`, for example set `target_rt_tolerance=0.0` for automatic retention time optimization.